CITATION.cff

```diff
@@ -5,6 +5,84 @@ identifiers:
 - type: doi
   value: 10.5281/zenodo.3555620
 authors:
+- given-names: Nicholas
+  family-names: Sofroniew
+  affiliation: Chan Zuckerberg Initiative
+  orcid: https://orcid.org/0000-0002-3426-0914
+  alias: sofroniewn
+- given-names: Talley
+  family-names: Lambert
+  affiliation: Harvard Medical School
+  orcid: https://orcid.org/0000-0002-2409-0181
+  alias: tlambert03
+- given-names: Grzegorz
+  family-names: Bokota
+  affiliation: University of Warsaw, Faculty of Mathematics, Informatics, and Mechanics
+  orcid: https://orcid.org/0000-0002-5470-1676
+  alias: Czaki
+- given-names: Juan
+  family-names: Nunez-Iglesias
+  affiliation: Monash eResearch Centre, Monash University
+  alias: jni
+- given-names: Peter
+  family-names: Sobolewski
+  affiliation: West Pomeranian University of Technology, Szczecin
+  orcid: https://orcid.org/0000-0002-2097-0990
+  alias: psobolewskiPhD
+- given-names: Andrew
+  family-names: Sweet
+  affiliation: Chan Zuckerberg Initiative
+  alias: andy-sweet
+- given-names: Lorenzo
+  family-names: Gaifas
+  affiliation: Gutsche Lab - University of Grenoble
+  orcid: https://orcid.org/0000-0003-4875-9422
+  alias: brisvag
+- given-names: Kira
+  family-names: Evans
+  affiliation: Chan Zuckerberg Initiative
+  alias: kne42
+- given-names: Alister
+  family-names: Burt
+  affiliation: MRC-LMB
+  alias: alisterburt
+- given-names: Draga
+  family-names: Doncila Pop
+  affiliation: Monash University
+  alias: DragaDoncila
+- given-names: Kevin
+  family-names: Yamauchi
+  affiliation: Iber Lab - ETH Zürich
+  alias: kevinyamauchi
+- given-names: Melissa
+  family-names: Weber Mendonça
+  affiliation: Quansight
+  orcid: https://orcid.org/0000-0002-3212-402X
+  alias: melissawm
+- given-names: Genevieve
+  family-names: Buckley
+  affiliation: Monash University
+  orcid: https://orcid.org/0000-0003-2763-492X
+  alias: GenevieveBuckley
+- given-names: Wouter-Michiel
+  family-names: Vierdag
+  affiliation: European Molecular Biology Laboratory, Genome Biology Unit, Heidelberg,
+    Germany
+  orcid: https://orcid.org/0000-0003-1666-5421
+  alias: melonora
+- given-names: Loic
+  family-names: Royer
+  affiliation: Chan Zuckerberg Biohub
+  alias: royerloic
+- given-names: Ahmet
+  family-names: Can Solak
+  affiliation: Chan Zuckerberg Biohub
+  alias: AhmetCanSolak
+- given-names: Kyle I. S.
+  family-names: Harrington
+  affiliation: Chan Zuckerberg Initiative
```

Comment:
@kephale Does this need to be updated to CZII?

Reply:
Long story, but it does not need to be updated. Thanks for checking 🙏